import_email_events.py

```diff
@@ -0,0 +1,17 @@
+#
+# Script to import "email event" metrics from S3 into redshift.
+#
+
+import import_events
+SCHEMA = """
+    type VARCHAR(30) NOT NULL,
+    locale VARCHAR(40),
+    domain VARCHAR(40),
+    template VARCHAR(64),
+    flow_id VARCHAR(64)
```

Comment:
One good rule of thumb I've found for choosing a distkey is, "if you were going to do a `COUNT DISTINCT` query on this table, which column would you be counting?".  I'm not sure there's an obvious answer here, but `flow_id` seems to come the closest.